core/SymbolRef.h

```diff
@@ -973,20 +973,32 @@ class Symbols {
         return ClassOrModuleRef::fromRaw(82);
     }
 
+    static MethodRef T_Private_Methods_DeclBuilder_abstract() {
+        return MethodRef::fromRaw(4);
+    }
+
+    static MethodRef T_Private_Methods_DeclBuilder_overridable() {
+        return MethodRef::fromRaw(5);
+    }
```

Comment:
These are currently unused, but I anticipate using them to implement the opposite feature, where you can click on these to go to the implementations of the method, so I figured I'd add them while I was making a mess to these ref numbers anyways.